core/src/main/java/org/aerogear/mobile/core/metrics/MetricsService.java

```diff
@@ -31,13 +31,13 @@ public String type() {
     }
 
     @Override
-    public void configure(MobileCore core, ServiceConfiguration serviceConfiguration) {
+    public void configure(final MobileCore core, final ServiceConfiguration serviceConfiguration) {
         defaultMetrics = new Metrics[]{
             new AppMetrics(core.getContext()),
```

Comment:
@pb82 This will lead to a NullPointerException if `core` os `serviceConfiguration` is null